### ISSUE TITLE
Bump Bump brick/math from ^0.10 to ^0.11

### ISSUE THREAD

**Issue by LocalHeroPro** [ramsey/uuid#500]
Right now `ramsey/uuid` required `"brick/math": "^0.8.8 || ^0.9 || ^0.10"`.
When I run:
```shell
$ php composer.phar outdated
Info from https://repo.packagist.org: #StandWithUkraine
//
Transitive dependencies not required in composer.json:
brick/math 0.10.2 0.11.0 Arbitrary-precision arithmetic library
```
So it's time to update brick/math to ^0.11.

**Comment by nclavaud** [comment]
Done with https://github.com/ramsey/uuid/pull/488, but not yet released!

**Comment by ramsey** [comment]
This is released in version 4.7.4. Thank you for your patience.
